admin/errors.go

```diff
@@ -0,0 +1,34 @@
+package admin
+
+import (
+	"errors"
+	"fmt"
+)
+
+var ErrValidatorReqDataFormat error = errors.New("wrong input format: expected JSON")
+
+type InvalidAdminParameterError struct {
```

Comment:
for custom error types, we usually implement this convenience method: 
https://github.com/onflow/flow-go/blob/93680d1b7ad29944d05f85f3f36e85289cff2a0b/consensus/hotstuff/committees/leader/leader_selection.go#L28-L31

Reply:
👍